src/drop-terms.test.js

```diff
@@ -0,0 +1,88 @@
+import assert from 'node:assert/strict';
+import createDebugLogger from 'debug';
+import fetchMock from 'fetch-mock';
+
+import validatorFactory from './drop-terms.js';
+
+import {MarcRecord} from '@natlibfi/marc-record';
+import {READERS} from '@natlibfi/fixura';
+import generateTests from '@natlibfi/fixugen';
+
+
+import {fakeData} from '../test-fixtures/translate-terms-data.js';
+
+
+const uris = [
+  'http://www.yso.fi/onto/yso/p444',
+  'http://www.yso.fi/onto/yso/p500',
+  'http://www.yso.fi/onto/yso/p13299',
+  'http://www.yso.fi/onto/yso/p111739',
+  'http://www.yso.fi/onto/yso/p6197061979',
+  'http://www.yso.fi/onto/yso/p6196061969',
+  'http://urn.fi/URN:NBN:fi:au:slm:s161'
+];
+
+const useMock = 1;
+
+
+generateTests({
+  callback,
+  path: [import.meta.dirname, '..', 'test-fixtures', 'drop-terms'],
+  useMetadataFile: true,
+  recurse: false,
+  fixura: {
+    reader: READERS.JSON
+  },
+  hooks: {
+    before: async () => {
+
+      if (useMock) {
+        fetchMock.mockGlobal()
```

Comment:
Replace with "native" mock to ensure longtime compatibility
https://github.com/nodejs/undici
https://nodejs.org/en/learn/test-runner/mocking#apis